django/template/context.py

```diff
@@ -245,12 +245,12 @@ def new(self, values=None):
         return new_context
 
 
-def make_context(context, request=None):
+def make_context(context, request=None, **kwargs):
     """
     Create a suitable Context from a plain dict and optionally an HttpRequest.
     """
     if request is None:
-        context = Context(context)
+        context = Context(context, **kwargs)
```

Comment:
It seems this only handles the `request=None` case; what about when a request is passed in?